x-pack/plugins/infra/server/services/log_entries/queries/log_entries.ts

```diff
@@ -52,8 +52,8 @@ export const createGetLogEntriesQuery = (
           ],
         },
       },
-      // @ts-expect-error @elastic/elasticsearch doesn't declare body.fields on AsyncSearchSubmitRequest
       fields,
+      // @ts-expect-error @elastic/elasticsearch expects "runtime" property
```

Comment:
Does it? I can find neither `runtime_mappings` nor `runtime` on the type. :thinking:

Reply:
Agree, I can't find it anymore. I will remove the comment